source/Editor.js

```diff
@@ -2106,7 +2106,7 @@ var keyHandlers = {
         var caretAtStartOfNode = selection.anchorOffset === 0;
 
         // Test is SPAN and has mathjax class.
-        var isMathjaxIE = !!(validNodes && anchorPreviousNode.matchesSelector && anchorPreviousNode.matchesSelector('span.mathjax'));
+        var isMathjaxIE = !!(validNodes && anchorPreviousNode.msMatchesSelector && anchorPreviousNode.msMatchesSelector('span.mathjax'));
```

Comment:
looks like you need `webkitMatchesSelector` for iOS 7: http://caniuse.com/#search=matches